src/services/functionAppService.ts

```diff
@@ -234,15 +207,20 @@ export class FunctionAppService extends BaseService {
 
     this.log(`-> Deploying service package @ ${functionZipFile}`);
 
+    const isLinux = this.configService.getOs() === FunctionAppOS.LINUX;
+    const uri = `https://${scmDomain}/api/zipdeploy${isLinux ? "?syncTriggers=true" : ""}`
+    this.log("Publishing to URI: " + uri);
+
     // https://github.com/projectkudu/kudu/wiki/Deploying-from-a-zip-file-or-url
     const requestOptions = {
       method: "POST",
-      uri: `https://${scmDomain}/api/zipdeploy/`,
+      uri,
       json: true,
       headers: {
         Authorization: `Bearer ${await this.getAccessToken()}`,
         Accept: "*/*",
         ContentType: "application/octet-stream",
+        "User-Agent": "serverless_framework", //TODO Underscore version name of plugin/cli
```

Comment:
Lets make sure we get this TODO into the backlog to add version number at some point in the near future.

Reply:
Added a note to the Roadmap